app/modules/fcm/partials/create.html

```diff
@@ -163,7 +155,7 @@ <h3 ng-if="mode == 'edit'"><span class="glyphicon glyphicon-pencil"></span> Edit
                     <table class="table table-hover">
                         <tr>
                             <td align="center" width="20%"><b># of Iteration</b></td>
-                            <td align="center" ng-repeat="n in range(1,totalIteration,1)"><b>{{ n }}</b></td>
+                            <td align="center" ng-repeat="n in range(1,,1)"><b>{{ n }}</b></td>
```

Comment:
@belalakbar range(1,,1) will generate syntax error. i guess it should be range(1,totalIteration,1), correct?